doc/Reference/Markup/Resources.md

```diff
@@ -0,0 +1,141 @@
+---
+uid: Reference.Markup.Resources
+---
+# Resource Dictionaries
+
+All Framework Elements and the Application have a couple of special extensions to help you manage your resources.
+
+## Merging Resource Dictionaries
+
+To merge resource dictionaries you simply need to add a Resource Dictionary to the Application or Framework Element like:
+
+```cs
+new Page()
+	// Where MyColorResources & MyFontResources are both a ResourceDictionary
+	.Resources(r => r.Merged(
+		new MyColorResources(),
+		new MyFontResources()))
+```
+
+## Adding Resources
+
+To add a resource to the Resource Dictionary you can use the fluent builder like:
+
+```cs
+new Page()
+	.Resources(resources => resources
+		.Add("MyColor", Colors.Red)
+		.Add("MyBrush", new SolidColorBrush(Colors.Red))
+)
+```
+
+### Resource Conversions
+
+There may be times when it may be difficult to provide a resource as the correct type. For example, you may want to provide a HEX string for a given color, but not want to convert the HEX values to bytes to create the color. The Markup library will use the built in converters to attempt to convert the supplied value to the correct type. For these sorts of resources it is recommended that you use the generic overload of the Add method which will convert the resource at the time it is created in the Resource Dictionary rather than each time it is used.
+
+```cs
+new Page()
+	.Resources(resources => resources
+		.Add<Color>("MyColor", "#FF0000") // Converted on Add ONLY
+		.Add("MyOtherColor", "#676767") // Converted EVERY time it is used
+	)
+	.Content(new TextBlock()
+		.Foreground(new SolidColorBrush()
+			.Color(StaticResource.Get<Color>("MyColor"))))
+```
+
+> [!TIP]
+> Existing XAML based projects may be using string resources for the path of an Icon. When using Markup be sure to convert this to a Geometry resource rather than leaving it a string. For example: `resources.Add<Geometry>("MyIcon", "M 0 0 L 10 10");`
```

Comment:
```suggestion
> Existing XAML based projects may be using string resources for the path of an `Icon`. When using Markup be sure to convert this to a Geometry resource rather than leaving it a string. For example: `resources.Add<Geometry>("MyIcon", "M 0 0 L 10 10");`
```